db/view/row_locking.cc

```diff
@@ -8,7 +8,6 @@
 
 #include "row_locking.hh"
 #include "log.hh"
-#include "utils/latency.hh"
```

Comment:
you're not adding latency tracking, you're just refactoring the implementation, right?

Reply:
then change $subject

Reply:
done